app/assess/templates/macros/scores.html

```diff
@@ -10,7 +10,7 @@
           </p>
         {% endif %}
         <div class="govuk-radios" data-module="govuk-radios">
-            {% for score, description in score_tuple %}
+            {% for score, description in score_list %}
```

Comment:
My worry is if `score_list` is `None` then we'll get a `TypeError` but not sure if it being a jinja2 template guards against that, if it's an empty collection it'll just skip over it.

Reply:
I'm managing this by doing an if score_list block in the jinja template, and as the logic is pythonic None is falsey so will just not display the score history stuff rather than errorring out which is ideal behavior!